### PR TITLE
[EXT-934] Some grammar and formatting changes

### PR DESCRIPTION
**Pull request by JeremyLawsonReach**

### CODE REVIEW

File: dropin/AUTOCAPTURE.md [hunk 1]
```diff
@@ -185,9 +187,10 @@ Instead, it indicates the payment was successfully initiated. You should use the
 the payment as complete in your system.
 
 ## Step 3 - Receive Notifications
-Lastly you will need to configure your server to listen for notifications. To do this an endpoint will need to be
-configured for the webhook URL provided to Reach during on-boarding. For more information on the notifications please
-Notifications section below.
+Lastly you will need to configure your server to listen for notifications. In order to receive
+these notifications you will need to configure your server to listen for them. To do this an endpoint will need to be
```
Comment (sandybrydon) [inline dropin/AUTOCAPTURE.md:191]: these 2 sentences are very similar

File: dropin/AUTOCAPTURE.md [hunk 2]
```diff
@@ -206,5 +209,5 @@ The `ORDER_PROCESSED` notification will look like:
 }
 ```
 
-Once successfully received a notification respond with a HttpStatus of 2XX to Reach. If a notification is replied to
-with a status outside that range, Reach will resend the notification at a later time.
+Once successfully received a notification respond with an HTTP status in the 200 range to Reach. If a notification is
```
Comment (sandybrydon) [inline dropin/AUTOCAPTURE.md:212]: this sentence is difficult
perhaps: Upon successful receipt of a notification respond with an HTTP status in the 200 range.
(or just "respond with HTTP status 200")

File: dropin/AUTOCAPTURE.md (hunk 1 above)
Comment (JeremyLawsonReach) [reply dropin/AUTOCAPTURE.md:191]: Feels like I just left for lunch in the middle of editing that part?

File: dropin/AUTOCAPTURE.md (hunk 2 above)
Comment (JeremyLawsonReach) [reply dropin/AUTOCAPTURE.md:212]: Yeah actually I like just telling them to respond with a 200

File: dropin/DELAYEDCAPTURE.md [hunk 3]
```diff
@@ -172,19 +175,21 @@ Putting this all together your page should look something like:
...
-provided to Reach during on-boarding.
+The Reach Drop-In will send notifications to you once actions are taken against a Session or Order. In order to receive
+these notifications you will need to configure your server to listen for them. To do this an endpoint will need to be 
+created by you at the webhook URL provided to Reach during on-boarding. For more information on the notifications please
+the [Notifications section](#/notifications/README.md).
 
-You will receive a `SESSION_COMPLETED` and `ORDER_AUTHORIZED` notification. Once receiving the `ORDER_AUTHORIZED` 
-notification you can mark the Order as ready to be captured in your system.
+You will receive a `SESSION_COMPLETED` and `ORDER_AUTHORIZED` notification once the user has submitted their payment. 
+Once  the `ORDER_AUTHORIZED` notification is received you can mark the Order as ready to be captured in your system.
```
Comment (sandybrydon) [inline dropin/DELAYEDCAPTURE.md:192]: "Once  the" -> "Once the"

File: dropin/README.md [hunk 4]
```diff
@@ -84,13 +86,28 @@ Please see the documents under [Notifications](/notifications/README.md) for mor
...
 ## Session State Diagram
+
+Sessions represent the user experience through a Drop-In transaction. The following diagram outlines the various transitions
+that a session can take, from instantiation to completion or failure.
+
 ![Session State Diagram](/assets/images/State%20Transition%20-%20Session%20States.jpeg)
 
 ## Order State Diagram
+
+When a session is moved to a COMPLETED state, an Order is created to represent the flow of funds for a Drop-In transactions. The
```
Comment (sandybrydon) [inline dropin/README.md:110]: `a Drop-In transactions` -> `a Drop-In transaction`

File: reachjs/LOCALIZATIONTEXT.md [hunk 5]
```diff
@@ -7,16 +7,20 @@
...
 ## Custom Text
 
-The Drop-In displays all text using text keys. It is possible for the parent page to override a limited set of these keys with different text. It may be useful to change the wording of certain elemements to be more in line with branding or phrasing when referring to certain things.
+The Drop-In displays all text using text keys. It is possible for the parent page to override a limited set of these 
+keys with different text. It may be useful to change the wording of certain elemements to be more in line with branding 
+or phrasing when referring to certain things.
 
-NOTE: Passing an override will supercede the user's provide locale for translated text. Take care to make sure any override text will be appropriate for the user's curent locale.
+NOTE: Passing an override will supercede the user's provide locale for translated text. Take care to make sure any 
+override text will be appropriate for the user's curent locale.
```
Comment (sandybrydon) [inline reachjs/LOCALIZATIONTEXT.md:23]: `curent` -> `current`

**Review by sandybrydon — APPROVED** [review]

File: dropin/README.md [hunk 6]
```diff
@@ -75,7 +77,7 @@ publicly accessible areas such as GitHub, client-side code, and so forth.**
 
 ## Notifications
 As part of the on-boarding for the Session API, you will provide Reach a webhook URL. This is where you should
-expect the notifications from Reach to be sent via a Http POST request. After successfully receiving a notification,
+expect the notifications from Reach to be sent via a HttP POST request. After successfully receiving a notification,
```
Comment (sandybrydon) [inline dropin/README.md:80]: HttP -> HTTP / Http

File: dropin/README.md [hunk 7]
```diff
@@ -45,7 +47,7 @@ Before you can accept payments with the Reach Drop-in solution you’ll need to
 - Complete the merchant on-boarding process. When completed you’ll receive:
   - A merchant ID (MID)
   - An HMAC secret
-- Confirm that you’ll be using the “Drop In” integration. When completed you’ll have setup:
+- Confirm that you’ll be using the Drop In integration. When completed you’ll have setup:
```
Comment (sandybrydon) [inline dropin/README.md:50]: `Drop In` -> `Drop-In`

**Review by sandybrydon — APPROVED** [review]

**Review by sandybrydon — APPROVED** [review]

**Review by tdawatson — APPROVED** [review]

**Pull request merged**